packages/compass-e2e-tests/helpers/commands/select-option.ts

```diff
@@ -26,9 +37,11 @@ export async function selectOption(
   await selectList.waitForDisplayed();
 
   // click the option
-  const optionSpan = selectList.$(`span=${optionText}`);
-  await optionSpan.scrollIntoView();
-  await optionSpan.click();
+  const option = optionText
```

Comment:
I know this is existing, but you can also browser.clickVisible(option), I think. That does scroll and various waits and more.